### PR TITLE
Fix heroku set command

### PR DESCRIPTION
**Pull request by duboff**
# What's up

The regex we were using to parse the set command only accepted word characters so it missed everything with a . for example. 

Solves #7

### CODE REVIEW

File: src/scripts/heroku-commands.coffee [hunk 1]
```diff
@@ -131,8 +131,10 @@ module.exports = (robot) ->
       listOfKeys = configVars && Object.keys(configVars).join(", ")
       respondToUser(msg, error, listOfKeys)
 
-  robot.respond /heroku config:set (.*) (\w+)=(\w+)/i, (msg) ->
+  robot.respond /heroku config:set (.*) (\w+)=(.*)/i, (msg) ->
```
Comment (matteodepalo) [inline src/scripts/heroku-commands.coffee]: Should we set the final regexp as `(.+)` so it doesn't match stuff like `heroku config:set appName key=`?

Comment (duboff) [inline src/scripts/heroku-commands.coffee]: @matteodepalo Good idea. I don't think we ever want the variable to be an empty string.

**Comment by matteodepalo** [comment]
:+1:

**Comment by daemonsy** [comment]
This looks good. Thanks @duboff for taking this on! 

Can I suggest to use this chance to add a few more test cases to the spec? 

I looked at some production apps' ENV variables. 

A few typical cases: 

Credentials encoded URIs: 
http://c0d9g8najfcc4634kd3:cf2eeas0d8dghfa847d@api.pusherapp.com/apps/1234

UUIDs: 
d5126f0d-b3be-46af-883f-b330a73964f9

RSA secret keys: 
----BEGIN RSA PRIVATE KEY-----\nsfsdfdssfdsFDSFDGSDfsdfsfs\nSDfSDFdUbOfFRocKsSFDSFSDFDS=\n-----END RSA PRIVATE KEY-----\n

Comma delimited strings: 
MiD, DA, MDe

If we can hit these cases, I think it's pretty robust :smile:

File: src/scripts/heroku-commands.coffee [hunk 2]
```diff
@@ -131,8 +131,10 @@ module.exports = (robot) ->
       listOfKeys = configVars && Object.keys(configVars).join(", ")
       respondToUser(msg, error, listOfKeys)
 
-  robot.respond /heroku config:set (.*) (\w+)=(\w+)/i, (msg) ->
+  robot.respond /heroku config:set (.*) (\w+)=(.+)/i, (msg) ->
     keyPair = {}
+
```
Comment (daemonsy) [inline src/scripts/heroku-commands.coffee]: Blank lines?

**Pull request closed without merging**